modules/onerepo/src/core/generate/commands/generate.ts

```diff
@@ -121,18 +119,13 @@ export interface Config<T extends Answers = Record<string, unknown>> {
 	prompts?: QuestionCollection<T>;
 }
 
-async function loadConfig(configPath: string) {
-	let config: Config | null;
-	if (configPath.endsWith('.cjs')) {
-		const require = createRequire('/');
-		config = require(configPath);
-	} else {
-		const { default: importConfig } = await import(configPath);
-		config = importConfig;
-	}
+const jiti = initJiti(process.cwd(), { interopDefault: true });
```

Comment:
is there a reason jiti is seemingly always iniitlaized at the top level of a file instead of within the method its used? does the cwd change if process.cwd is called within a function?

Reply:
It can be done either place. I'm not sure if it's necessarily less performant at the top level vs function scope.

The path given as the first argument sets where relative `requires` would be pulled from. So if you set `process.cwd()` and try to run `jiti('./foo')` it will be relative to `process.cwd()`. In some places, `import.meta.url` or `__filename` are used so that imports are relative to the local file.